helm/prometheus-rules/templates/alerting-rules/inhibit.management-cluster.rules.yml

```diff
@@ -59,7 +59,6 @@ spec:
         cluster_status_deleting: "true"
         team: phoenix
         topic: status
-    {{- if eq .Values.managementCluster.provider.kind "aws" }}
```

Comment:
wow, just noticed that the only vintage provider we support is now aws! :exploding_head:

Reply:
I did a lot of work around that last week https://github.com/giantswarm/prometheus-rules/blob/master/CHANGELOG.md#removed